### PR TITLE
Get data from mongodb and add product props pushing to mongodb 

### PR DESCRIPTION
**Pull request by l-goede**

### CODE REVIEW

**Review by J-Bannach — APPROVED** [review]
lgtm 👍

File: src/app/components/StaticWatchlistProducts/DroppedPriceProduct.module.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  padding: 0;
+  margin: 0;
+  max-width: 80%;
+}
+
+.droppedPriceProduct__description {
+  color: var(--primary-color);
+  font-size: 0.8em;
+  margin-top: 0;
+  margin-bottom: 0.25em;
```
Comment (lajanzen) [inline src/app/components/StaticWatchlistProducts/DroppedPriceProduct.module.css:56]:
```suggestion
margin: 0 0 0.25em
```

File: src/app/components/StaticWatchlistProducts/StaticWatchlistProduct.module.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.staticWatchlistProduct__icon:hover {
+  cursor: pointer;
+}
+
+.staticWatchlistProduct__title {
+  font-size: 0.9em;
+  padding: 0;
+  margin: 0;
```
Comment (lajanzen) [inline src/app/components/StaticWatchlistProducts/StaticWatchlistProduct.module.css:49]: Do you need this?

**Review by lajanzen — APPROVED** [review]
🚀

**Pull request merged**